packages/vouching/test/helpers/expectEvent.js

```diff
@@ -0,0 +1,33 @@
+const should = require('chai').should();
+
+function inLogs (logs, eventName, eventArgs = {}) {
+  const event = logs.find(function (e) {
+    if (e.event === eventName) {
+      let matches = true;
+
+      for (const [k, v] of Object.entries(eventArgs)) {
+        if (e.args[k] !== v) {
+          matches = false;
+        }
+      }
+
+      if (matches) {
+        return true;
+      }
```

Comment:
Yes, I know. I took the methods from OpenZeppelin's repo testing suite, and seemed right to keep it the same instead of changing the code for this minor thing.